src/ApiTests/ApiTests.csproj

```diff
@@ -10,11 +10,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="ApprovalTests" Version="3.0.13" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
-    <PackageReference Include="NUnit" Version="3.8.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
-    <PackageReference Include="PublicApiGenerator" Version="6.5.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
```

Comment:
This needs to stay at 3.8.0-alpha1 for now because of conflicts with Mono.Cecil

Reply:
Seems to work fine thought on the CI server

Reply:
Right, this is about making sure it works correctly in Visual Studio and other places that would actually use the test adapter.